x-pack/plugins/security_solution/cypress/tasks/alerts_detection_rules.ts

```diff
@@ -132,16 +133,19 @@ export const deleteRuleFromDetailsPage = () => {
 };
 
 export const duplicateSelectedRules = () => {
+  cy.log('Duplicate selected rules');
```

Comment:
Thank you for this improvement 👍 

Nit: Can we / should we do it like this to support refactorings easier:

```ts
cy.log(duplicateSelectedRules.name)
```

The downside is probably slightly worse readability of Cypress logs.

Reply:
We'll also need to rewrite all helpers to named functions for this to work. But I'd prefer log messages to be more easily readable by humans as the test runner's output is already overloaded with technical information.